rust/agents/relayer/src/processor.rs

```diff
@@ -0,0 +1,37 @@
+use std::fmt::Debug;
+
+use async_trait::async_trait;
+use derive_new::new;
+use eyre::Result;
+use hyperlane_core::HyperlaneDomain;
+use tokio::task::JoinHandle;
+use tracing::{info_span, instrument, instrument::Instrumented, Instrument};
+
+#[async_trait]
+pub trait ProcessorExt: Send + Debug {
```

Comment:
Ext == extension? not familiar with this naming convention

Reply:
Still in RFC but fairly common
https://rust-lang.github.io/rfcs/0445-extension-trait-conventions.html